sample.csv

```diff
@@ -0,0 +1,4 @@
+1,Slivered Almonds:22.88;Wholewheat flour:1.93;Grape Seed Oil:74.9
+2,Albacore Tuna:36.92;Capers:97.99;Sultanas:2.82;Koshihikari rice:7.55
```

Comment:
I love the idea of making a smaller file that's easier to work with for your tests! This is a technique that we'll explore more once we get into Rails in a few weeks.